src/main/java/uk/gov/hmcts/sscs/service/RoboticsService.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.sscs.service;
+
+import java.io.InputStream;
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.sscs.json.RoboticsJsonGenerator;
+import uk.gov.hmcts.sscs.model.robotics.RoboticsWrapper;
+
+@Service
+public class RoboticsService {
+
+    private final InputStream inputStream;
+
+    private final Schema schema;
+
+    public RoboticsService() {
+        inputStream = getClass().getResourceAsStream("/schema/sscs-robotics.json");
```

Comment:
This isn't a big thing I guess, but I don't think constructors should be used for anything other than accepting required dependencies and setting them into properties.
https://www.google.co.uk/search?q=constructors+should+not+do+work

Also, I would prefer the json generator to be a dependency injected into this coordinating class -- static methods bind calling code to a concrete implementation in an inflexible way and are not well suited to a growing project.

Finally, I would prefer the schema validator to be extracted out and dependency injected into this coordinating class too.